### PR TITLE
Add Debian curl packages and dependencies

### PR DESCRIPTION
**Pull request by GiasemiSh**

### CODE REVIEW

**Comment by hemarkus** [comment]
[INFO] Connecting targets to packages
2019/10/01 16:49:17 Connect failed: rpc error: code = Internal desc = grpc: failed to unmarshal the received message proto: can't skip unknown wire type 4
make: *** [Makefile:33: demos/debian-curl] Error 1

File: demos/debian-curl/build.sh [hunk 1]
```diff
@@ -22,12 +22,20 @@ echo "master server up and running"
 
 echo "[INFO] Create packages"
 qmstrctl create package:curl_7.64.0-3_amd64.deb --version $(cd curl && git describe --tags --dirty --long)
+qmstrctl create package:libcurl4_7.64.0-3_amd64.deb
```
[the remark this answers is not in the document]
Comment (GiasemiSh) [reply demos/debian-curl/build.sh]: Updated it.

**Comment by hemarkus** [comment]
Fails with:

[INFO] Connecting targets to packages
2019/10/04 12:47:23 Connect failed: rpc error: code = Internal desc = grpc: failed to unmarshal the received message proto: can't skip unknown wire type 4
make: *** [Makefile:33: demos/debian-curl] Error 1

**Comment by GiasemiSh** [comment]
> Fails with:
> 
> [INFO] Connecting targets to packages
> 2019/10/04 12:47:23 Connect failed: rpc error: code = Internal desc = grpc: failed to unmarshal the received message proto: can't skip unknown wire type 4
> make: *** [Makefile:33: demos/debian-curl] Error 1

You are not using the updated qmstr commands

**Pull request merged**